marketplace-client/src/marketplace/src/app/product-catalog/add-product/add-product.component.ts

```diff
@@ -82,21 +109,22 @@ export class AddProductComponent {
   onSubmit(): void {
     this.submitted = true;
     if (this.form.invalid) {
+      console.log(this.form.value);
+      console.log("dont work");
```

Comment:
don't commit error logs. If you want to notify a user, make a notification on the site